config.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version='1.0' encoding='utf-8'?>
-<widget android-versionCode="541" id="org.phidatalab.radar_armt" ios-CFBundleIdentifier="org.phidatalab.radar-armt" ios-CFBundleVersion="5" version="0.7.3" xmlns:android="http://schemas.android.com/apk/res/android">
+<widget android-versionCode="542" id="org.phidatalab.radar_armt" ios-CFBundleIdentifier="org.phidatalab.radar-armt" ios-CFBundleVersion="1" version="0.7.4" xmlns:android="http://schemas.android.com/apk/res/android">
```

Comment:
why is the ios version change back to 1? Is it due to the production release?

Reply:
In iOS, for the same version, the bundle version has to be unique, but for different versions, they can repeat. I was thinking to start back from `1` since this is a new version, so that it's easier to differentiate alpha and production?

Reply:
Ok Sure. Sounds good.